index.js

```diff
@@ -1,17 +1,62 @@
-const fs = require('fs');
-const path = require('path');
-
 module.exports = {
-  bundle: {
-    directory: 'modules',
-    modules: getBundleModuleNames()
+  extend: '@apostrophecms/piece-type',
+  options: {
+    label: 'aposSvgs:label',
+    pluralLabel: 'aposSvgs:pluralLabel',
+    i18n: {
+      ns: 'aposSvgs',
+      browser: true
+    },
+    // Opt out of open graph, SEO, search, and quick create
+    openGraph: false,
+    seoFields: false,
+    searchable: false,
+    quickCreate: false,
+    // Auto-publish (no draft state)
```

Comment:
Actually, you will need to set it `localize: false` if you are going to use a mongodb `update` for the update case, as otherwise the autopublish mechanism has no idea an update is happening and can't publish it.

I think that probably makes sense. But if it doesn't — for instance because the titles may need localization — then you can keep `autopublish: true` if you switch to using `self.update` rather than a mongodb $set operation.

Reply:
I think we should leave the door open for folks to localize titles. These are icons, so it probably not often necessary, but it could help with alt text.